examples/standalone_mode_demo.ipynb

```diff
@@ -0,0 +1,609 @@
+    "\n",
+    "cmd_args = {\n",
+    "    'yaml_config_file': yaml_config_file,\n",
+    "    'part_config': constructed_graph_config,\n",
+    "    'local_rank': os.environ['RANK'],\n",
+    "}\n",
+    "cmd_args = Namespace(**cmd_args)\n",
+    "config = GSConfig(cmd_args)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "INFO:root:Start to load partition from /mnt/efs/gsf-data/movielen_100k_train_val_1p_4t/part0/graph.dgl which is 5839353 bytes. It may take non-trivial time for large partition.\n",
+      "INFO:root:Finished loading partition.\n",
+      "INFO:root:Finished loading node data.\n",
+      "INFO:root:Finished loading edge data.\n",
+      "INFO:root:part 0, train: 1345, val: 168, test: 168\n"
+     ]
+    }
+   ],
+   "source": [
+    "# load the constructed graph data\n",
+    "train_data = GSgnnNodeTrainData(\n",
+    "    graph_name=config.graph_name,\n",
+    "    part_config=config.part_config,\n",
+    "    train_ntypes=config.target_ntype,\n",
+    "    eval_ntypes=config.target_ntype,\n",
+    "    node_feat_field=config.node_feat_name,\n",
+    "    label_field=config.label_field)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Number of nodes:\n",
+      "- movie: 1682\n",
+      "- user: 943\n",
+      "Number of edges:\n",
+      "- rating-rev: 100000\n",
+      "- rating: 100000\n"
+     ]
+    }
+   ],
+   "source": [
+    "# train_data.g stores a DistGraph object\n",
+    "print('Number of nodes:')\n",
+    "for ntype in train_data.g.ntypes:\n",
+    "    print(f'- {ntype}: {train_data.g.num_nodes(ntype)}')\n",
+    "print('Number of edges:')\n",
+    "for etype in train_data.g.etypes:\n",
+    "    print(f'- {etype}: {train_data.g.num_edges(etype)}')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# set up data loaders\n",
+    "dataloader = GSgnnNodeDataLoader(\n",
+    "    train_data, train_data.train_idxs, fanout=config.fanout,\n",
+    "    batch_size=config.batch_size,\n",
+    "    device=device, train_task=True)\n",
+    "\n",
+    "val_dataloader = GSgnnNodeDataLoader(\n",
+    "    train_data, train_data.val_idxs, fanout=config.fanout,\n",
+    "    batch_size=config.batch_size,\n",
+    "    device=device, train_task=False)\n",
+    "\n",
+    "test_dataloader = GSgnnNodeDataLoader(\n",
+    "    train_data, train_data.test_idxs, fanout=config.fanout,\n",
+    "    batch_size=config.batch_size,\n",
+    "    device=device, train_task=False)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 2. Create a GNN model\n",
```

Comment:
Could you try to use the code in the `gs.create_builtin_node_gnn_model()` to build a model, and check what GSConfig arguments will be used? I think this is the place where many GSConfig arguments are used, and need to be replaced.

Also, we expose the encoder and gnn layer APIs upon which users can use to build their own customer models. In such case, we better reveal the details of create_builtin_node_gnn_model().